pkg/storage/config/store.go

```diff
@@ -47,14 +41,8 @@ func (cfg *ChunkStoreConfig) RegisterFlags(f *flag.FlagSet) {
 	cfg.WriteDedupeCacheConfig.RegisterFlagsWithPrefix("store.index-cache-write.", "", f)
 
 	f.Var(&cfg.CacheLookupsOlderThan, "store.cache-lookups-older-than", "Cache index entries older than this period. 0 to disable.")
-	f.Var(&cfg.MaxLookBackPeriod, "store.max-look-back-period", "This flag is deprecated. Use -querier.max-query-lookback instead.")
 }
 
 func (cfg *ChunkStoreConfig) Validate(logger log.Logger) error {
-	if cfg.MaxLookBackPeriod > 0 {
```

Comment:
nit: remove this method and it's usages?

Reply:
thought about that. But left it as it as to be consistent with other configs. 

We check all the configs validate [here](https://github.com/grafana/loki/blob/kavirajk%2Fremove-store-look-back-period/pkg/loki/loki.go#L247-L249)